.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
What is this for exactly?

Reply:
I followed husky from the site and it got created. So I didn't touch it

Reply:
Cool. After your PR, I think we will also need to instruct users to run `npm run prepare` after `npm i` to set up this hook. So add that instruction in the README.